template_snippets/cloudconfig/g8s_master.yaml

```diff
@@ -101,6 +107,8 @@
       -v /etc/kubernetes/cni/:/etc/kubernetes/cni/ \
       -v /opt/cni/bin/calico:/opt/cni/bin/calico \
       -v /opt/cni/bin/calico-ipam:/opt/cni/bin/calico-ipam \
+      -v /usr/bin/mount.quobyte:/sbin/mount.quobyte \
+      -v /usr/bin/umount.quobyte:/sbin/umount.quobyte \
```

Comment:
this is needed, because `kubelet` test if it can support  storage by testing if it has available mount binaries for that type of file system